benchmark/build.gradle.kts

```diff
@@ -43,6 +43,11 @@ android {
             isDefault = true
         }
     }
+
+    lint {
+        warningsAsErrors = true
+        lintConfig = file("${project.rootDir}/config/lint/lint.xml")
+    }
```

Comment:
This is true @wzieba , but then again, during development, while I was running `./gradlew lintDebug` to verify things, Lint kept failing for me. Actually, I kept hitting that problem for months now, but never bother to do anything about it. Now, and since the configuration was very straightforward, I wanted to fix so that we Lint never fails again for this project, no matter the module.